main/UseCases/GameGenerators.java

```diff
@@ -118,15 +120,19 @@ public void generateFinalProject() throws FileNotFoundException {
      * @param skillList an ArrayList of skills.
      * @return a HashMap of Skills and the score (0.00 - 1.00) of the skill
      */
-    private HashMap<String, Double> generateUniqueSkillMap(ArrayList<String> skillList){
```

Comment:
Should keep as private, please do change back to private.

Reply:
sure!